application/modules/upload/controllers/Upload.php

```diff
@@ -122,7 +122,7 @@ public function show_files($url_key, $customerId = null)
                     $obj['name'] = substr($file, strpos($file, '_', 1) + 1);
                     $obj['fullname'] = $file;
                     $obj['size'] = filesize($path . '/' . $file);
-                    $obj['fullpath'] = $path . '/' . $file;
+					$obj['fullpath'] = $file; // Full path removed to as the patch for CVE-2021-29022
```

Comment:
A few weeks ago we decided to get rid of the entire filepath #739. This PR was already merged into `develop`.
It seems to me, that you want to merge into `master`, maybe that's the issue?

Thanks nevertheless for this contribution!

Reply:
Make sense, then we don't need to merge this.